staging/volumes/rbd/rbd.yaml

```diff
@@ -22,6 +22,4 @@ spec:
         readOnly: true
         user: admin
         keyring: /etc/ceph/keyring
-        imageformat: "2"
```

Comment:
@lincappu please, provide more details on why this change is needed.

Also, please sign the CLA.